adafruit_ht16k33/segments.py

```diff
@@ -289,12 +289,13 @@ def _number(self, number: Union[int, float], decimal: int = 0) -> str:
                 places += 1
 
         # Set decimal places, if number of decimal places is specified (decimal > 0)
+        txt = None
         if places > 0 < decimal < len(stnum[places:]) and dot > 0:
             txt = stnum[: dot + decimal + 1]
         elif places > 0:
             txt = stnum[:places]
 
-        if len(txt) > self._chars + 1:
+        if txt is None or len(txt) > self._chars + 1:
```

Comment:
If text is None, the error message bout it being too long doesn't make sense.

Reply:
With the change of `txt = stnum` this `None` guard can go away, I'll remove it.